tests/ManageCourses.Tests/DbIntegration/AdminControllerTests.cs

```diff
@@ -0,0 +1,101 @@
+
+using System.Collections.ObjectModel;
+using System.Linq;
+using FluentAssertions;
+using GovUk.Education.ManageCourses.Api.Controllers;
+using GovUk.Education.ManageCourses.Domain.Models;
+using GovUk.Education.ManageCourses.Tests.DbIntegration;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+
+namespace GovUk.Education.ManageCourses.Tests.Controllers
+{
+    [TestFixture]
+    public class AdminControllerTests : DbIntegrationTestBase
+    {
+        [Test]
+        public void ActionAccessRequest_HappyPath()
```

Comment:
So what's the happy path? Adding or updating?

Reply:
Can we update to something more professional?

Reply:
Where the test for the `unhappy path`?

Reply:
👍